lib/gitlab_core.ml

```diff
@@ -1478,6 +1486,21 @@ struct
         API.post ~token ~uri ~body ~expected_code:`Created (fun s ->
             Lwt.return (Gitlab_j.project_access_token_of_string s))
     end
+
+    module Issue = struct
+      let list ?token ~project_id () =
+        let uri = URI.project_issues project_id in
+        API.get_stream ?token ~uri (fun body -> return (Gitlab_j.issues_of_string body))
```

Comment:
Don't you want to force the user to give the `token` here?

Reply:
In this case no, you can still list the issues on a project without a token, you'll get the publicly available issues and a worse rate limit on HTTP requests.